Test/ghost/elseWithHavocStatement.dfy

```diff
@@ -0,0 +1,13 @@
+// RUN: %baredafny run %args --target=py %s > %t
+// RUN: %testDafnyForEachCompiler "%s"
+
+method Main() {
+    var b: bool :| true; 
+    var r: int;
+    if b {
+      r := 13;
+    } else {
+      r := *;
+    }
+    print r;
+}
```

Comment:
`Test/comp/Various.dfy` might be a good fit. The whole problem is that this isn't ghost code, right?

Reply:
Why would we put it in another file? `Various` doesn't tell me much. I would rather put it in a file related to `If` statements.

Reply:
I prefer fewer files with multiple tests, because they make browsing the directory easier. I think you should at least put the file somewhere else. The `ghost` directory seems wrong.

Reply:
Moved it to a sensible directory

Reply:
I don't think the tests are closely related. The others test the verifier, or the resolver, but not the compilers, for which we have a dedicated folder.

Reply:
What are you suggesting we do then?

I have placed the new test in `comp/` and left the existing tests unchanged.